OpenOversight/app/main/forms.py

```diff
@@ -46,6 +46,7 @@ class FindOfficerForm(Form):
     unique_internal_identifier = StringField('unique_internal_identifier', default='', validators=[Regexp(r'\w*'), Length(max=55)])
     dept = QuerySelectField('dept', validators=[DataRequired()],
                             query_factory=dept_choices, get_label='name')
+    unit = SelectField('unit', default='Not Sure')
```

Comment:
You are using a `SelectField` without providing choices. This is what leads to the error with `test_user_can_use_form_to_get_to_browse` (although this certainly should be surfaced better)
Since the options for unit are not static like race and gender and furthermore depend on the department that was selected, I think the right choice would be to do whatever the solution for `rank` is, since that field has similar constraints (I am not familiar with how this is done though)